### ISSUE TITLE
Arc Radius is only in meters

### ISSUE THREAD

**Issue by kylekarwatski** [mpetroff/qgsazimuth#18]
The arc radius for a segment appears to be in meters always. Unlike distance, there is no setting for radius units. Some users may assume that setting distance units to feet applies to the radius as well.

When plotting from a data set that I have [that is in US Standard units: feet], an arc was drawn very shallow. I manually calculated and created some arc segments. Then I converted my feet to meters for the radius of my segment and the result lined up with my manual attempt.

Maybe you can add a separate setting for radius units or even generalize the unit setting.

**Comment by mpetroff** [comment]
The unit selection not being honored for the arc radius input was an oversight. The person who wrote the patch that added arc drawing support doesn't have the misfortune of having to deal with US customary units, and I never noticed that it didn't work properly with feet.

I fixed the plugin so when the feet option is selected, it also applies to the arc radius. This will be in v0.9.12, which I've uploaded to the plugin repository (it's currently awaiting approval).
